inc/classes/ServiceProvider/class-third-party-subscribers.php

```diff
@@ -33,6 +33,7 @@ class Third_Party_Subscribers extends AbstractServiceProvider {
 		'ewww_webp_subscriber',
 		'optimus_webp_subscriber',
 		'bigcommerce_subscriber',
+		'amp_subscriber',
```

Comment:
Since we are touching this file, let's move it into `inc/ThirdParty/ServiceProvider.php` and rename the class to  `ServiceProvider`.

Reply:
done in https://github.com/wp-media/wp-rocket/pull/2459/commits/218a81f02ceb1fce826e9c5c11519975f4174a47